src/I18n/Number.php

```diff
@@ -276,6 +276,10 @@ public static function formatter($options = [])
             $type = NumberFormatter::CURRENCY;
         }
 
+        if (!empty($options['type']) && $options['type'] === 'ordinal') {
```

Comment:
It's actually `int` for all types, I don't know there's a special case for currency.